packages/styles/scss/components/background-media/background-media.scss

```diff
@@ -0,0 +1,88 @@
+//
```

Comment:
Let's rename the file to `_background-media.scss` to keep it consistent with the other `scss` partial within this `styles` package directory.